scripts/release-weex.sh

```diff
@@ -20,15 +21,15 @@ if [[ $REPLY =~ ^[Yy]$ ]]; then
   WEEX_VERSION=$NEXT_VERSION npm run build:weex
 
   # update package
-  cd packages/weex-vue-framework
-  npm version $NEXT_VERSION
+  ( cd packages/weex-vue-framework
+  npm version "$NEXT_VERSION"
   npm publish
-  cd -
+  )
 
-  cd packages/weex-template-compiler
-  npm version $NEXT_VERSION
+  ( cd packages/weex-template-compiler
+  npm version "$NEXT_VERSION"
   npm publish
-  cd -
+  )
```

Comment:
This is a nice trick! Though I'm not sure if it'd be clear for everyone what is it doing. I'd stick with the `cd -` just for clarity.

Reply:
I can add a comment like: `using a subshell to avoid having to cd back`.

It's not just to avoid writing a line, it also can prevent some troubles, see [SC2013](https://github.com/koalaman/shellcheck/wiki/SC2103).

Reply:
Oh, I didn't know about that one! It should be modified then. The comment would be helpful :)

Reply:
Done.